src/GraphRenderer.ts

```diff
@@ -297,7 +297,8 @@ export class GraphRenderer {
 
     static portDragSuggestedNode : ko.Observable<Node> = ko.observable(null);
     static portDragSuggestedField : ko.Observable<Field> = ko.observable(null);
-    static matchingPortList : {field:Field,node:Node}[] = []
+    static portDragSuggestionValidity : ko.Observable<Errors.Validity> = ko.observable(Errors.Validity.Unknown) // this is necessary because we cannot keep the validity on the ege as it does not exist
+    static matchingPortList : {field:Field,node:Node,validity: Errors.Validity}[] = []
```

Comment:
Not your fault, since this was the existing variable name. But, like we were discussing last week, I think it is good practice to remove "type" information from the variable names.

Better to call this "matchingPorts" instead of "matchingPortList". No worries if you want to leave it though, its not a change you made.